test-data/pretty_format_yaml/multi-doc-not-pretty-formatted.yaml

```diff
@@ -0,0 +1,9 @@
+---
+root:
+    test:
+     -  1
```

Comment:
I would make this doc very ugly, so the result will be more visible
```suggestion
root: {test: [1]}
```

Reply:
I can make it more ugly, but your suggestion is already pretty-formatted according to the tool.

Reply:
That's surprising ... but yeah. Let's ignore this comment.
I'm marking this as resolved